### PR TITLE
NTP: Update images mentioned as having been over-compressed

### PR DESCRIPTION
**Pull request by cezaraugusto**
Fix https://github.com/brave/brave-browser/issues/6904
Fix https://github.com/brave/brave-browser/issues/7997

This PR includes new images provided in https://github.com/brave/brave-browser/issues/6904#issuecomment-573851691 without any extra change. Please reach out to the design team if there are concerns about image compression/quality. This PR also reverts https://github.com/brave/brave-core/pull/4357, making https://github.com/brave/brave-core/pull/4264 available again.

### CODE REVIEW

File: components/brave_new_tab_ui/data/LICENSE [hunk 1]
```diff
@@ -1,63 +1,59 @@
...
+dc-cavalleri.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-9.webp:
-URL: https://www.instagram.com/p/BI7w_dFAlR0/
-License: used with permission
+joe-gardner.webp:
+URL: https://unsplash.com/photos/4xv3lqnanYc
+License: Used with permission
```
Comment (fmarier) [inline components/brave_new_tab_ui/data/LICENSE]: This should be https://unsplash.com/license.

File: components/brave_new_tab_ui/data/LICENSE [hunk 2]
```diff
@@ -1,63 +1,59 @@
...
+louis-kim.webp:
+URL: Contributor sent the hi-res version through email
+License: Used with permission
 
-ntp-11.webp:
-URL: https://www.instagram.com/p/BNyaAuFBiSt/
-License: used with permission
+matt-palmer.webp:
+URL: https://unsplash.com/photos/veMnvjmfoxw
+License: Used with permission
```
Comment (fmarier) [inline components/brave_new_tab_ui/data/LICENSE]: This should be https://unsplash.com/license.

Comment (cezaraugusto) [reply components/brave_new_tab_ui/data/LICENSE]: done

File: components/brave_new_tab_ui/data/LICENSE (hunk 1 above)
Comment (cezaraugusto) [reply components/brave_new_tab_ui/data/LICENSE]: done

**Comment by bsclifton** [comment]
This introduced a problem with licensing; CI wasn't checked before merge. It's holding up Nightly. Will need to revert

**Comment by bsclifton** [comment]
Revert abandoned; this WAS properly fixed with https://github.com/brave/brave-core/pull/4864 😄 When build problem happened, it was an older version (1.7.48) which didn't have this fix. Images are good to go and should be in 1.7 and newer!

**Pull request merged**